src/main/java/chopchop/logic/parser/commands/FilterCommandParser.java

```diff
@@ -139,8 +139,8 @@
 
             if (s.isEmpty()) {
                 return Optional.of(String.format("%s cannot be empty", kind));
-            } else if (s.split("\\s+").length > 1) {
-                return Optional.of(String.format("Filter terms should not contain spaces"));
+            } else if (kind == "Expiry date" && s.split("\\s+").length > 1) {
```

Comment:
this should be `kind.equals("Expiry date")`

Reply:
Okie will change

Reply:
Okie changed, thankss

Reply:
Thanks, it's updated now